ewfs/noise_models.py

```diff
@@ -1,22 +1,32 @@
 """Custom noise models for simulator-based experiments."""
+from typing import Set
+
+from qiskit import QuantumCircuit
+
 from qiskit_aer.noise import (
     depolarizing_error,
     NoiseModel,
     pauli_error,
 )
 
 
-def depolarizing_noise_model(error: float = 0.01) -> NoiseModel:
+def depolarizing_noise_model(single_qubit_error_rate: float=0.01, two_qubit_error_rate: float=0.03, 
```

Comment:
[nit]: Given that the default of a type set can be `None`, here is a somewhat more explicit way to type this:

```suggestion
def depolarizing_noise_model(single_qubit_error_rate: float=0.01, two_qubit_error_rate: float=0.03, 
                             single_qubit_gates: set | None = None, two_qubit_gates: set | None = None) -> NoiseModel:
```